lib/src/css_mirroring/rulesets_processor.dart

```diff
@@ -29,7 +29,7 @@ RemovalResult editFlippedRuleSet(MirroredEntity<RuleSet> mirroredRuleSet,
     Direction flippedDirection, BufferedTransaction trans) {
   final subTransaction = trans.createSubTransaction();
 
-  MirroredEntities<Declaration> mirroredDeclarations = mirroredRuleSet
+  MirroredEntities<RuleSet> mirroredDeclarations = mirroredRuleSet
```

Comment:
This change seems wrong, declarations is a `List<Declaration>` (see comment above re/ making getChildren generic)